wiki/Welcome/ru.md

```diff
@@ -28,22 +24,24 @@ outdated: true
 
 *Подробнее: [Маппинг](/wiki/Beatmapping).*
 
-Маппинг — это процесс создания карты. Он включает в себя выбор музыки, настройку тайминга, расстановку игровых элементов и тестирование карты. Также создатель карты может добавить видео, [сториборд](/wiki/Storyboarding) или свой [скин](/wiki/Skinning). Всё это может быть сделано им как самостоятельно, так и совместно с другими пользователями.
+Маппинг — это процесс создания игровой карты (битмап). Он включает в себя выбор музыки, настройку тайминга, расстановку игровых элементов и тестирование карты. К тому же, создатель может включить в карту видео, [сториборд](/wiki/Storyboarding) и/или свой [скин](/wiki/Skinning). Всё это может быть сделано им как самостоятельно, так и совместно с другими пользователями, которых иногда называют по типу проделанной ими работы.
 
 ## Моддинг
 
 *Подробнее: [Моддинг](/wiki/Modding).*
 
-Моддинг — это просмотр и улучшение новой карты другими участниками (которых обычно называют моддерами). Моддинг играет важную роль в создании качественных карт, поскольку он помогает найти и исправить ошибки.
+Моддинг — это просмотр и улучшение новой карты другими участниками (которых обычно называют моддерами). Моддинг играет важную роль в создании качественных карт, поскольку он помогает создателям находить и исправлять ошибки в своих картах.
 
 ## Скины
 
-*Подробнее: [Скины](/wiki/Skinning) и [Учебник по скинам](/wiki/Skinning/Tutorial).*
+*Подробнее: [Скины](/wiki/Skinning)
+
+*И [Учебник по скинам](/wiki/Skinning/Tutorial).
 
-Скины позволяют каждому изменить то, как osu! выглядит и ощущается. Можно внести как небольшие правки (например поменять курсор) так и полностью переделать интерфейс игры и оформление во всех режимах.
+Скины позволяют каждому изменить то, как osu! выглядит и ощущается. Можно внести как небольшие правки, например, изменить курсор, так и полностью переделать интерфейс игры и её оформление во всех режимах.
 
-## osu! wiki - нам нужна ваша помощь!
+## osu! wiki - Как вы можете нам помочь!
 
-Начиная с 2016-08-26 osu! wiki была перенесена в репозиторий на GitHub. После этого рабочий процесс сильно изменился. Все правки теперь вносятся через [репозиторий osu-wiki на GitHub](https://github.com/ppy/osu-wiki) и проверяются перед тем, как они будут  опубликованы на osu! wiki.
+Начиная с 26-08-2016 osu! wiki была перенесена в репозиторий на GitHub. В связи с этим, рабочий процесс сильно изменился. Все правки теперь вносятся через [репозиторий osu-wiki на GitHub](https://github.com/ppy/osu-wiki) и проверяются перед тем, как они будут  опубликованы на osu! wiki.
```

Comment:
this date can be written in a more natural format (like "26 August, 2016", however that is in Russian)